model_utils/tracker.py

```diff
@@ -40,10 +39,7 @@ def __get__(self, instance, owner):
         if instance is None:
             return self
         was_deferred = self.field_name in instance.get_deferred_fields()
-        try:
-            value = self.descriptor.__get__(instance, owner)
-        except AttributeError:
-            value = self.descriptor
+        value = self.descriptor.__get__(instance, owner)
```

Comment:
```suggestion
        try:
            value = self.descriptor.__get__(instance, owner)
        except AttributeError:
            value = self.descriptor
```

Reply:
Hey, this just reverts my changes?
I've already forgotten a lot of what was going on here, but I believe the try-catch was only a bandaid fix that caused other edge cases to fail.
The real fix, I believe, needs to be made in django. See my open ticket and PR with them here: https://code.djangoproject.com/ticket/30427#ticket
Helping get that merged in would, then allow us to fix the issue properly here.

Reply:
Ok thank you for your advice.